devicetypes/smartthings/open-closed-sensor.src/open-closed-sensor.groovy

```diff
@@ -40,14 +40,12 @@ metadata {
 
 // Parse incoming device messages to generate events
 def parse(String description) {
-	def name = null
-	def value = description
-	if (zigbee.isZoneType19(description)) {
-		name = "contact"
-		value = zigbee.translateStatusZoneType19(description) ? "open" : "closed"
+	def resMap
+	if (description.startsWith("zone status")) {
```

Comment:
I guess you're handling this in other places by just checking `description.startsWith("zone")` which should work too.